tests/e2e/tests/admin/transfer-tokens.spec.ts

```diff
@@ -57,9 +60,13 @@ test.describe('Transfer Tokens', () => {
 
     await navToHeader(page, ['Settings', 'Transfer Tokens'], 'Transfer Tokens');
 
-    const row = page.getByRole('gridcell', { name: 'my test token', exact: true });
-    await expect(row).toBeVisible();
-    await expect(page.getByText(/\d+ (second|minute)s? ago/)).toBeVisible();
+    const nameCell = page.getByRole('gridcell', { name: 'my test token', exact: true });
+    await expect(nameCell).toBeVisible();
+
+    // Locate the parent of nameCell and then search within it for the timestamp
+    const parentRow = nameCell.locator('xpath=..');
+    await expect(parentRow).toContainText(/\d+ (second|minute)s? ago/);
```

Comment:
Since we aren't clearing the db after each run anymore and have multiple tokens, we need to be more specific with what we're looking for